decidim-accountability/app/forms/decidim/accountability/admin/result_import_projects_form.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Accountability
+    module Admin
+      # A form object to be used when admin users want to import a collection of projects
+      # from another component into Accountability component.
+      class ResultImportProjectsForm < Decidim::Form
+        attribute :origin_component_id, Integer
+        attribute :import_all_selected_projects, Boolean
+
+        validates :origin_component_id, presence: true
+        validates :import_all_selected_projects, allow_nil: false, acceptance: true
+        validates :to_be_added_projects, numericality: { greater_than: 0 }, if: ->(form) { form.origin_component_id }
+
+        def origin_component
+          @origin_component ||= origin_components.find_by(id: origin_component_id)
+        end
+
+        def origin_components_collection
+          origin_components.map do |component|
+            [component.name[I18n.locale.to_s], component.id]
+          end
+        end
+
+        def origin_components
+          @budgets_component ||= current_participatory_space.components.where(manifest_name: "budgets")
+        end
+
+        def to_be_added_projects
```

Comment:
This is a bit different from projects count, since this the number of projects that are selected to be imported and are not copied yet. So I would suggest something like not_copied_projects, or pending_projects

Reply:
OK, I think the `to_be_` prefix just doesn't sound very Ruby-like.

Maybe we can rename it to `origin_projects_count`?

I understand that it is counting the `selected` projects but the only place where I see this being used is this form class itself. Maybe you can make the method private when the method would not be exposed outside of this class not to confuse anyone about it?